.github/workflows/actions.yml

```diff
@@ -39,6 +39,8 @@ jobs:
             coverage lcov -o /etc/conbench-coverage-dir/coverage.lcov
       - name: Publish coverage
         uses: coverallsapp/github-action@master
+        # sometimes the coveralls API fails transiently, and we don't want that to affect this build
+        continue-on-error: true
```

Comment:
I kept running into failures here so I added this to this PR. I could split it out if you wish.